index_test.go

```diff
@@ -210,7 +210,7 @@ func searchForTest(t *testing.T, b *IndexBuilder, q query.Q, o ...SearchOptions)
 	return res
 }
 
-func searcherForTest(t *testing.T, b *IndexBuilder) Searcher {
+func searcherForTest(t *testing.T, b *IndexBuilder, mux ...func(searcher MuxSearcher)) Searcher {
```

Comment:
just make this return a MuxSearcher instead rather than passing in a list of functions. We can do this since NewSearcher returns MuxSearcher

Reply:
New latest commit removes this code. I setting this convo to resolved.